### DISCUSSION TITLE
Does this project support running on X86 32-bit windows system?

### DISCUSSION THREAD

**Discussion by ChdDongyang** [devopvoid/webrtc-java#83] [Q&A]
I ran the project under 32-bit windows system and got the following error:

`Caused by: java.lang.UnsatisfiedLinkError: C:\Users\Administrator\AppData\Local\Temp\webrtc-java1154973935403791856.dll: Can't load AMD 64-bit .dll on a IA 32-bit platform`

Is there a substitute for the Dynamic Link Library that supports 32-bit systems?

Please help me.

**Comment by devopvoid** [comment]
Hi,
no support for 32 bit systems yet. Should not be an issue creating a 32 bit build pipeline for that.
But I feel like 32 bit is kind off obsolete. You can build this project by yourself, though.
